### PR TITLE
ci: Security Scans

### LINKED ISSUE #7951: Add security tools to the pipeline
To enhance security in Keptn, we could introduce a few tools that check for security best practices on a pipeline level.

Ideas:
- introduce tools in pipelines that are run manually
- introduce tools in pipelines that are run on a weekly basis and generate github issues from the findings/reports
- introduce tools that prevent PR merges
- In addition to security tools, also kubernetes/yaml best practice tools could be introduced

### Security
- [kubescape](https://github.com/armosec/kubescape)
- [kics](https://github.com/Checkmarx/kics)

### Best practices
- [kubeconform](https://github.com/yannh/kubeconform)
- [kubeval](https://github.com/instrumenta/kubeval)

### PR DESCRIPTION
**Pull request by mowies**
### This PR
- Adds a new pipeline to run security scans with the following tools:
  - KICS
  - kubescape
  - kubeconform
- the pipeline runs scheduled on mondays and can also be triggered manually

Fixes #7951

### CODE REVIEW

File: .github/kics-config.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+exclude-queries:
+  # query IDs can be found here: https://docs.kics.io/latest/queries/all-queries/
```
Comment (thisthat) [inline .github/kics-config.yml:2]: I would suggest for future self when revisiting this pipeline to add in the comment also the reason why we exclude the specific rule

File: .github/workflows/security-scans.yml [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        run: |
+          VERSION=$(cat './VERSION.txt')
+          cd ./installer/manifests/keptn
+          helm repo add bitnami https://charts.bitnami.com/bitnami
+          helm repo add nats https://nats-io.github.io/k8s/helm/charts/
+          helm dependency update ./
+          helm dependency build ./
+          # replace "appVersion: latest" with "appVersion: $VERSION" in all Chart.yaml files
+          find . -name Chart.yaml -exec sed -i -- "s/appVersion: latest/appVersion: ${VERSION}/g" {} \;
+          helm template ./ -n keptn --name-template keptn --version 0.17.0 > ${{ env.RENDERED_CHART_FILENAME }}
```
Comment (thisthat) [inline .github/workflows/security-scans.yml:37]: would it make sense to get the version from the latest semver tag?

File: .github/kics-config.yml (hunk 1 above)
Comment (mowies) [reply .github/kics-config.yml:2]: i will add a comment stating why those queries are excluded

File: .github/workflows/security-scans.yml (hunk 2 above)
Comment (mowies) [reply .github/workflows/security-scans.yml:37]: yes, i will fix that :)

**Review by thisthat — APPROVED** [review]

**Review by warber — APPROVED** [review]

**Pull request merged**